Server/Content/Sources/App/Controllers/TextController.swift

```diff
@@ -0,0 +1,29 @@
+import Fluent
+import Vapor
+
+struct TextController {
+    func all(req: Request) throws -> EventLoopFuture<[Text]> {
+        return Text.query(on: req.db).all()
+    }
+    
+    func getById(req: Request) throws -> EventLoopFuture<Text> {
```

Comment:
Texts are not really going to be retrieved by id, client-side (iOS app) doesn't even know that we have id for texts. It just needs a random text each time from us. 

You are designing API, do not forget that, you are not just writing some layer between database and client. You are designing an interface for client to communicate with. 

So if you respect our brothers from client-side, please keep in mind that you need to provide as convenient API for them as possible